cfpb/consumerfinance.gov
# Enable auto indexing for ES7

Enables auto indexing and auto syncing for our ES 7 indexes, except when running tests (so we don't require an ES7 server connected for testing).

---

## Changes

- Now only disable auto indexing during our tests.


## How to test this PR

1. Index one of our elasticsearch documents
2. Save/change a model that is stored in elasticsearch (Eg: Change test in an AnswerPage for ask-cfpb)
3. Confirm the updated content is present in ES7 after the save occurs.

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
